prow/config/jobs.go

```diff
@@ -149,6 +151,15 @@ type Postsubmit struct {
 
 	// TODO(krzyzacy): Move existing `Report` into `Skip_Report` once this is deployed
 	Reporter
+
+	JenkinsJob
+}
+
+// Optional Jenkins job config
+type JenkinsJob struct {
+	// Job is managed by the GH branch source plugin
+	// and requires a specific path
+	GithubBasedProject bool `json:"github_based_project,omitempty"`
```

Comment:
Definitely open to naming this differently.

Reply:
If this is only useful for a workflow provided by a specific plugin we should probably make that clear from the name so that users don't think this is something generally useful for projects hosted on GitHub.

Reply:
`GithubBranchSourcePlugin`, perhaps?